src/pages/home/Home.js

```diff
@@ -0,0 +1,88 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+import {Link, NavLink} from "react-router-dom";
+import logo from '../../assets/logo.png' ;
+import './Home.css';
+
+function Home() {
+    const [reddits, setReddits] = useState([]);
+    const [error, toggleError] = useState(false);
+    const [loading, toggleLoading] = useState(false);
+
+    useEffect(() => {
+        async function getData() {
+            toggleLoading(true);
+            try {
+                toggleError(false);
+                const result = await axios.get('https://www.reddit.com/hot.json?limit=15');
+                setReddits(result.data.data.children);
+                console.log(result.data.data.children);
+            } catch (e) {
+                console.error(e);
+                toggleError(true);
+            }
+            toggleLoading(false);
+        }
+
+        if (true) {
+            getData();
+        }
```

Comment:
Dit kun je versimpelen
```suggestion
        getData();
```